doc/migrate-to-pymongo4.rst

```diff
@@ -980,6 +980,13 @@ The default uuid_representation for :class:`~bson.codec_options.CodecOptions`,
 :data:`bson.binary.UuidRepresentation.PYTHON_LEGACY` to
 :data:`bson.binary.UuidRepresentation.UNSPECIFIED`. Attempting to encode a
 :class:`uuid.UUID` instance to BSON or JSON now produces an error by default.
+If you were using UUIDs previously, you will need to set your uuid_representation to
```

Comment:
uuid_representation should be in double-backticks.

Reply:
Yes, I think so

Reply:
Done.